test/integration/generated/nameChecker/src/models/parameters.ts

```diff
@@ -340,8 +340,8 @@ export const selectedFields: OperationQueryParameter = {
   collectionFormat: "CSV"
 };
 
-export const search$DONotNormalize$Text: OperationQueryParameter = {
-  parameterPath: "search$DONotNormalize$Text",
+export const searchDONOTNormalizeText: OperationQueryParameter = {
+  parameterPath: "searchDONOTNormalizeText",
```

Comment:
Maybe this is because I set the Style.camel(str, true, {}, 5) but if I don't set it as 5, I get a mismatch in bodyComplex of CMYKColor in models/index.ts.